es/13_3_Potenciando_Bitcoin_con_Scripts.md

```diff
@@ -0,0 +1,317 @@
+Running: <SigBob> <pubKeyBob> OP_CHECKSIG
+Stack: [ True ]
+```
+
+#### Ejecuta el script de bloqueo para una Alice virtuosa
+
+Todas las transacciones de compromiso de Alice están bloqueadas con este mismo script de bloqueo, ya sea que hayan sido revocadas o no. Eso significa que la transacción de compromiso más reciente, que es la válida actualmente, también está bloqueada. Alice nunca ha enviado una nueva transacción a Bob y, por lo tanto, nunca le envió el `revokeCode` anterior.
+
+En esta situación, podría publicar virtuosamente la transacción, cerrando el canal proto-Lightning. Ella pone la transacción en la cadena y espera 24 horas. Bob no puede hacer nada al respecto porque no tiene el código de recuperación. Luego, después de la espera, Alice reclama sus fondos. (Bob hace lo mismo con su propia transacción de compromiso final).
+
+### Bloqueo con contratos codificados con bloqueo de tiempo
+
+Las Transacciones de Compromiso Revocable fueron solo un trampolín hacia Lightning. La Lightning Network real utiliza un mecanismo más complejo llamado [contrato de bloqueo de tiempo hash] (https://en.bitcoin.it/wiki/Hashed_Timelock_Contracts), o HTLC.
+
+El objetivo principal de los HTLC es crear una red integral de participantes. Las transacciones ya no son solo entre un par de participantes que han ingresado a la red juntos, sino que ahora pueden ser entre personas previamente no asociadas. Cuando se envían fondos, se crea una serie de transacciones, cada una de ellas bloqueada con un "secretHash". Cuando se revela el "secretCode" correspondiente, se puede gastar toda la cadena de transacciones. Esto es lo que permite que las transacciones singulares se conviertan realmente en una red.
+
+También hay un poco más de complejidad en los scripts de bloqueo de Lightning Network. Hay bloqueos separados para el remitente y el destinatario de cada transacción que son más divergentes que las diferentes transacciones de compromiso mencionadas en la sección anterior. Vamos a mostrarlos a ambos para demostrar el poder de estos scripts de bloqueo, pero no vamos a detenernos en cómo interactúan entre sí.
+
+#### Bloquear la transacción del destinatario
+
+Una vez más, comenzaremos a ver la transacción de compromiso de Alice, que muestra los fondos que recibió:
+
+```
+OP_HASH160 
+OP_DUP 
+<secretHash> 
+OP_EQUAL
+    
+IF
+
+    <+24Hours>
+    OP_CHECKSEQUENCEVERIFY
+    OP_2DROP
+    <pubKeyAlice>
+    
+ELSE
+
+    <revokeHash> 
+    OP_EQUAL
+        
+    OP_NOTIF
+            
+        <Date> 
+        OP_CHECKLOCKTIMEVERIFY 
+        OP_DROP
+        
+    ENDIF
+        
+    <pubKeyBob>
+    
+ENDIF
+    
+OP_CHECKSIG
+```
+
+La clave de estos nuevos HTLC es el `secretHash`, que dijimos es lo que permite que una transacción abarque la red. Cuando la transacción se ha extendido desde su originador hasta su destinatario previsto, se revela el "secretCode", que permite a todos los participantes crear un "secretHash" y desbloquear toda la red de pagos.
```

Comment:
- **Replace** double quotes (` " `) in `"secretHash"` and `"secretCode"` **by** backticks (`` ` ``).